docu/files/SkeletonCreatorBuilderCLI.html.md

```diff
@@ -1,45 +1,65 @@
 ---
-title: 'Design a Builder UI with Skeleton Creator'
+title: 'Advanced Features for developers'
 service: 'Builder SDK'
 area: 'Core'
 type: Tutorial
 order: 3
 ---
 
-The Builder CLI features the Skeleton Creator tool, with which you can easily create views just by dragging and dropping UI components. <br><br>
-
-<h3>Run the Skeleton Creator</h3>
+<h3>Design a Builder UI with Skeleton Creator</h3>
 
+The Builder CLI features the Skeleton Creator tool, with which you can easily create views just by dragging and dropping UI components. Here are the steps to get started.<br>
 
 1. Open a terminal window.
 
- <img src="img/builder_cli_menu_runSkeletor.png"><br><br>
+<img style="width: 30%;" max-width: 326px src="img/builder_cli_menu_runSkeletor.png" class="img-click-modal" alt="Skeleton Creator"><br><br>
 
 2. Run the command `builder runSkeletonCreator` to start the Skeleton Creator. The HTTP server starts locally.
 3. Enter the URL <b>http://localhost:8082</b> in a browser.  The Skeleton Creator design interface will appear.
 4. To design a view, drag and drop the components from the left of the screen onto the main screen. Organize the components as they should appear in the Builder UI.
 
-  <img src="img/skeletor_ui_drag.png" style="width:600px" class="img-click-modal" alt="The Skeleton Creator UI"/><br><br>
-  5. After you create the view, click the preview button to the right of the delete button.  The browser immediately stores your view in the local cache. That means you will not loose the view until you delete it by explicitly clicking on the delete button.
+<img style="width: 30%;" max-width: 326px src="img/skeletor_ui_drag.png" class="img-click-modal" alt="The Skeleton Creator UI"/><br><br>
+
+5. After you create the view, click the preview button to the right of the delete button.  The browser immediately stores your view in the local cache. That means you will not loose the view until you delete it by explicitly clicking on the delete button.
```

Comment:
Also, are the preview and delete buttons labeled? I assume not, since there is no capitalization or bold formatting and you are using "the button." If that's a correct assumption, then great; just verifying. :)